index.js

```diff
@@ -1,25 +1,26 @@
 const express = require('express')
+
+const Dictionary = require('./src/dictionary')
+const Game = require('./src/game')
+
 const app = express()
-const Dictionary = require('./src/Dictionary')
-Dictionary.getWord().then(word => console.log('word: ', word))
+let wordGuess
+let id = 25
+
+Dictionary.getWord().then(word => wordGuess = word)
 
-app.get('/', (req, res) => res.send('Hello World!'))
+app.get('/', (req, res) => res.send(wordGuess))
 
 app.get('/game', (req, res) => {
-  res.send({
-    id: 1,
-    hint: '_____A',
-    remainings: 5,
-    image: `
- _______
-|/      |
-|      (_)
-|      \|/
-|       |
-|      / \\n
-|
-|___`
-  })
+  Game.create({id: id, guess: wordGuess})
```

Comment:
I think the game should generate the id and don't expected it as a parameter

Reply:
also the "wordGuess" should be generated in the game.create function, as it is implemented now there is only one word created for the whole game